### PR TITLE
Upgrade support for uproot 4.0.0

### PR DESCRIPTION
**Pull request by HealthyPear**
Just a negligible fix of an RST formatting I missed last time....

**EDIT:** during this we discovered that uproot made a new release that required less negligible changes, so the scope of the PR got superseded.

### CODE REVIEW

**Comment by HealthyPear** [comment]
@maxnoe weird: I only modified the README, but I get an error from the docs when executing the notebook.
It seems to be something related to uproot, but on my local branch I do not get this error...have you ever noticed it in previous PRs?

**Comment by maxnoe** [comment]
https://github.com/scikit-hep/uproot4/releases/tag/4.0.0 was just released, so maybe the API changed?

**Comment by HealthyPear** [comment]
Seems so.
Should we then "embrace" 4.0.0 event though they say it's a "non-release-candidate"?
If we agree on this, I will remove all `errors` key arguments from that method and simplify the imports, since now uproot means uproot4 by default

**Comment by maxnoe** [comment]
> Should we then "embrace" 4.0.0 event though they say it's a "non-release-candidate"?

Non-release-candidate means it is the first *real* release.

So yes, we should just require `uproot~=4.0` and do `import uproot` instead of `import uproot4 as uproot`

**Comment by HealthyPear** [comment]
Don't mind the last merge commit, I mistakenly pushed into an old branch...

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
         "pytest-cov",
         "gammapy~=0.18",
         "ogadf-schema~=0.2.3",
-        "uproot4",
+        "uproot",
```
Comment (maxnoe) [inline setup.py:23]: Please add the version requirement `~=4.0`

**Review by maxnoe — CHANGES_REQUESTED** [review]

**Review by maxnoe — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
         "pytest-cov",
         "gammapy~=0.18",
         "ogadf-schema~=0.2.3",
-        "uproot4",
+        "uproot~=4.0",
         "awkward1",
```
Comment (maxnoe) [inline setup.py:24]: Sorry missed that. `awkward1` should now also be `awkward~=1.0`

**Review by maxnoe — CHANGES_REQUESTED** [review]
awkward needs to be updated as well.

File: setup.py (hunk 2 above)
Comment (HealthyPear) [reply setup.py]: done

**Review by maxnoe — APPROVED** [review]

**Pull request merged**